misk/src/main/kotlin/misk/security/csp/Csp.kt

```diff
@@ -0,0 +1,15 @@
+package misk.security.csp
+
+// This annotation allows misk endpoints to define their Content-Security-Policy directive.
```

Comment:
nit: change the comment type to proper javadoc format (replace `//` with `/** */`)